src/app/registration/success/facilityConfirmation/diamondHead/diamondHeadConfirmation.component.html

```diff
@@ -52,7 +52,7 @@ <h5 class="mb-4">Safety information</h5>
             </div>
           </div>
         </div>
-
+  </div>
```

Comment:
I think the indenting got missed, but looks like it closes properly now.

Reply:
Resolved in [Commit](https://github.com/bcgov/parks-reso-public/pull/443/commits/3e554cc73896dab5f64267c13c4429b74c3a696b)